### PR TITLE
Portfolio setup

### PR DESCRIPTION
**Pull request by Yulianav09**
# Portfolio setup
* Used semantic HTML to create the header and the introduction for the portfolio
* Used Flexbox to arrange the elements on the page 
* Added font families, colors, and sizes according to the Figma file

### CODE REVIEW

**Review by Efiamotu-1 — COMMENTED** [review]
Hi <STUDENT_MENTION>, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request
 

You have submitted a project, that is missing key sections of the [readme file](https://github.com/microverseinc/readme-template) such as `key- features, future features, acknowlegements`. please ensure that sections relevant to your project are not missing from your readme when you are resubmitting for a new review :+1: 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+</head>
+<body>
+  <header>
+    <h1>My logo</h1>
+    <div id="hamburgerContainer">
+      <div class="hamburgerBar">&nbsp;</div>
+      <div class="hamburgerBar">&nbsp;</div>
+      <div class="hamburgerBar">&nbsp;</div>
```
Comment (Shedrack-Sunday) [inline index.html:17]: - [x] Kindly adjust the semantics of the HTML here on the hamburger to have s a `nav` tag.

- The `nav` tag is for semantics reasons, and the right position for convention and user experience for mobile devices.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    </div>
+  </header>
+  <main>
+    <section id="intro">
+      <h2>
+        Hey There.
+        I’m Yuliana.
+      </h2>
```
Comment (Shedrack-Sunday) [inline index.html]: - [x] As it stands as the main headline, kindly make it a `h1` tag.
- If you want that google (and other search engines) to know about a specific topic, in this case, you, use an `h1` tag. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info

File: index.html [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+      <p id="subtitle">I'm a Software Developer</p>
+      <p id="introduction">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded,
+        don’t hesitate to contact me.</p>
+        <div id="socialMedia">
+          <figure><img src="./src/github.svg" alt="Github icon"></figure>
+          <figure><img src="./src/Linkedin icon.svg" alt="Linkedin icon"></figure>
+          <figure><img src="./src/angel list.svg" alt="Angel list icon"></figure>
+          <figure><img src="./src/twitter.svg" alt="twitter icon"></figure>
+          <figure><img src="./src/medium.svg" alt="Medium icon"></figure>
```
Comment (Shedrack-Sunday) [inline index.html]: - [x] For this section, The list of icons should use a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag).

-  Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href.

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hello @Yulianav09     👋 

# CHANGES REQUIRED ♻️

Great work on the project 👏
You've done extremely well so far in this project,  However, there's one adjustment you need to make to move to the next project. 💯



## Highlights   
- Good use of GitHub features ✔️  
- Kudus on linter setup ✔️ 
- Nice design 👏 
  
# Required Changes  🌵 

- [x] You made some impressive implementations which I must say is a job well done. However, there are a few  I suggest   you make In line with the requirements 

_Check the comments for a detailed review 

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing really improves your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 2 above)
Comment (Yulianav09) [reply index.html]: Hello @Shedrack-Sunday, thanks for your comments.
I already used the H1 tag in the header as it is a common practice to use the logo or the initial part of the header as an H1, and according to best practices I should have only one H1 on the entire page, should I remove the H1 from the header or what is the best procedure in this case?
Regards

**Review by ridwanediallo — APPROVED** [review]
# STATUS: APPROVED 🟢
## Hi @Yulianav09 , 
Congratulations!GIF

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Well done implementing the changes requested by the previous reviewer. ✔
- You successfully completed all the requirements.  🎉 🎉 🎉 ✔

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**